build/tooling/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-bullseye
+FROM golang:1.19.4-bullseye
```

Comment:
can we use `golang@sha256:bb9811fad43a7d6fd2173248d8331b2dcf5ac9af20976b1937ecd214c5b8c383` for now? this is Go 1.19.5 without git change

Reply:
let's create an issue so we can track once there's a fix

Reply:
Pushed the change, also created this issue to track:

https://github.com/Azure/eraser/issues/630